### PR TITLE
Create the single choice component

### PR DESCRIPTION
**Pull request by diegoAndres01**
Create the single choice component

### CODE REVIEW

File: src/components/layout/SingleChoice/stories/SingleChoice.stories.tsx
Comment (Andresbl123) [inline src/components/layout/SingleChoice/stories/SingleChoice.stories.tsx:1]: Como recomendación, para solucionar el error en la historia del componte appPege, y el error de de la tipografía en el contenido del `input`, es de crear dos archivos en el directorio `.storybook`,  `preview-head.html` : `<link rel="preconnect" href="https://fonts.googleapis.com" />
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
<link
  href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
  rel="stylesheet"
/>` y   `preview-body.html` : `<div id="portals"></div>`

**Review by Andresbl123 — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -2569,358 +2570,6 @@
       "resolved": "https://registry.npmjs.org/@emotion/unitless/-/unitless-0.8.1.tgz",
       "integrity": "sha512-KOEGMu6dmJZtpadb476IsZBclKvILjopjUii3V+7MnXIQCYh8W3NgNcgwo21n9LXZX6EDIKvqfjYxXebDwxKmQ=="
     },
-    "node_modules/@esbuild/aix-ppc64": {
```
Comment (carin91) [inline package-lock.json:2572]: Why do you delete this dependency? i'm unable to run storybook due to the lack of this dependency.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: .env [hunk 2]
```diff
@@ -1,9 +0,0 @@
-VITE_USERS_URI=http://localhost:3000/privileges/users
```
Comment (carin91) [inline .env:1]: Please do not delete the .env, i will talk with Dora Lucia to solve this issue.

File: src/components/inputs/SingleChoice/index.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+    const handleSelect = (event: React.ChangeEvent<HTMLInputElement>, name: string) => {
+      setValueSelect(event.target.innerText);
+      handleSelectChange(event, name);
+    }
+        
+
+  return (
+    <>
+        <Toggle
```
Comment (carin91) [inline src/components/inputs/SingleChoice/index.tsx:53]: I would create a new component to wrap the others (single choice, multiple choice and so on)

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/components/inputs/SingleChoice/index.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+export const SingleChoice = (props: SingleChoiceProps) => {
+    const {
+        handleSelectChange,
+        id,
+        labelSelect,
+        name,
+        options,
+        required,
```
Comment (carin91) [inline src/components/inputs/SingleChoice/index.tsx:24]: avoid the coercion using required=false

**Review by carin91 — APPROVED** [review]

**Pull request merged**